### PR TITLE
Fix label street

### PR DESCRIPTION
**Pull request by azime**

### CODE REVIEW

File: src/bin/osm2mimir.rs [hunk 1]
```diff
@@ -234,6 +234,12 @@ fn get_street_admin(admins_geofinder: &AdminGeoFinder,
     coord.map_or(vec![], |c| admins_geofinder.get(&c))
 }
 
+fn format_label(admins: &AdminsVec, city_level: u32, name: &String) -> String {
```
Comment (TeXitoi) [inline src/bin/osm2mimir.rs]: name should be a `&str`

File: src/bin/osm2mimir.rs [hunk 2]
```diff
@@ -234,6 +234,12 @@ fn get_street_admin(admins_geofinder: &AdminGeoFinder,
...
+    match admins.iter().position(|adm| adm.level == city_level) {
+        Some(idx) => format!("{} ({})", name, admins[idx].label),
+        None => (*name).clone()
```
Comment (TeXitoi) [inline src/bin/osm2mimir.rs]: I think the star operator is not needed

File: tests/osm2mimir_test.rs [hunk 3]
```diff
@@ -46,7 +46,7 @@ pub fn osm2mimir_sample_test(es_wrapper: ::ElasticSearchWrapper) {
                         &es_wrapper);
     // Test: Import of Admin
     let res: Vec<_> = es_wrapper.search_and_filter("label:Livry-sur-Seine", |_| true).collect();
-    assert_eq!(res.len(), 1);
+    assert_eq!(res.len(), 4);
     assert!(res[0].is_admin());
```
Comment (TeXitoi) [inline tests/osm2mimir_test.rs]:
```
res.iter().any(|r| r.is_admin())
```

File: tests/osm2mimir_test.rs [hunk 4]
```diff
@@ -55,26 +55,26 @@ pub fn osm2mimir_sample_test(es_wrapper: ::ElasticSearchWrapper) {
     assert!(res.len() != 0);
     assert!(res[0].is_street());
     // The first hit should be "Rue des Près"
-    assert!(res[0].label() == "Rue des Près");
+    assert!(res[0].label() == "Rue des Près (Livry-sur-Seine)");
 
     // And there should be only ONE "Rue des Près"
-    assert_eq!(res.iter().filter(|place| place.is_street() && place.label() == "Rue des Près").count(), 1);
-    
+    assert_eq!(res.iter().filter(|place| place.is_street() && place.label() == "Rue des Près (Livry-sur-Seine)").count(), 1);
```
Comment (TeXitoi) [inline tests/osm2mimir_test.rs]: long line

File: src/bin/osm2mimir.rs (hunk 1 above)
Comment (azime) [inline src/bin/osm2mimir.rs]: OK

**Comment by TeXitoi** [comment]
OK for me.

**Pull request merged**